Automattic/wp-calypso
# Reader: Use the same styles for post comments as post content

#### Changes proposed in this Pull Request

* Use the same we use to render post content in reader to also render post comments

I just copied and pasted the styles but obviously they should be put in a shared location... I'm just not sure of where that should be. Any suggestions?

#### Testing instructions



* Load a post with some comments left using the block editor (for example `/read/feeds/104773847/posts/2656283156`) and ensure that headings, tables, quotes and other blocks all have the expected styles applied to them
